Formula/tsduck.rb

```diff
@@ -0,0 +1,43 @@
+class Tsduck < Formula
+  desc "MPEG Transport Stream Toolkit"
+  homepage "https://tsduck.io/"
+  url "https://github.com/tsduck/tsduck/archive/v3.29-2651.tar.gz"
+  sha256 "cab8f5838993aa1abd1a6a4c2ef7f2afba801da02a4001904f3f5ba5c5fe85a0"
+  license "BSD-2-Clause"
+  head "https://github.com/tsduck/tsduck.git", branch: "master"
+
+  depends_on "dos2unix" => :build
+  depends_on "gnu-sed" => :build
+  depends_on "grep" => :build
+  depends_on "openjdk" => :build
+  depends_on "librist"
+  depends_on "srt"
+  uses_from_macos "curl"
+  on_linux do
+    depends_on "gcc" => :build
+    depends_on "libedit"
+    depends_on "pcsc-lite"
+  end
```

Comment:
```suggestion
  uses_from_macos "curl"
  uses_from_macos "libedit"
  uses_from_macos "pcsc-lite"
```
Homebrew GCC isn't actually used unless you declare `fails_with`, and will use the host GCC otherwise.

That seems to have worked, so we can just omit it entirely.

Reply:
Does `uses_from_macos "xxx"` imply `on linux depends_on "xxx" end`?
Or shall we keep the `on_linux ...`?